core-build/gulp-core-build/src/tasks/jest.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "title": "Jest Task configuration",
+  "description": "Defines Jest task configuration",
+
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "description": "Part of the JSON Schema standard, this optional keyword declares the URL of the schema that the file conforms to. Editors may download the schema and use it to perform syntax highlighting.",
+      "type": "string"
+    },
+    "ci": {
+      "description": "Indicate whether the test is running in a CI environment",
+      "type": "boolean"
+    },
+    "configFilePath": {
+      "description": "The path to a jest config file",
+      "type": "string"
+    },
+    "coverage": {
+      "description": "Indicates that test coverage information should be collected and reported in the output",
+      "type": "boolean"
+    },
+    "maxWorkers": {
+      "description": "Specifies the maximum number of workers the worker-pool will spawn for running tests.",
+      "type": "number"
+    },
+    "rootDir": {
+      "description": "The root directory for the project",
```

Comment:
>"The root directory for the project", [](start = 21, length = 37)

This is standardized right?  We should not need to specify it for each project

Reply:
Yeah this probably shouldn't be configurable.

---
In reply to: [144159911](https://github.com/Microsoft/web-build-tools/pull/391#discussion_r144159911) [](ancestors = 144159911)

Reply:
Removed.

---
In reply to: [144161141](https://github.com/Microsoft/web-build-tools/pull/391#discussion_r144161141) [](ancestors = 144161141,144159911)